src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -103,11 +105,12 @@ function CustomizedInputBase(props) {
           alignItems="center"
           justify="flex-start"
         >
-          <Grid item xs={12}>
```

Comment:
Why this change?

Reply:
Leaving that in wouldn't allow me to put the <SearchHelp> beside the header since it has built in css with xs={12}

Reply:
OK, sounds good.